src/plugins/LogicGatesExporter/metadata.json

```diff
@@ -1,14 +0,0 @@
-
-{
```

Comment:
Did you uninstall this or just deleted the files? 
The `webgme-setup.json` might still contain it and the `config.webgme.js`.
Also make sure it's not referenced in the seed..